### ISSUE TITLE
Blockchain Document Store

### ISSUE THREAD

**Issue by saicharanpogul** [IBM/Medical-Blockchain#5]
Hello,
I emailed Rak-Joon Choi (rak-joon.choi@us.ibm.com) but didn't get any reply yet its been almost a week.

**Comment by saicharanpogul** [comment]
?

**Comment by pinoycoach** [comment]
Thank you for reaching out, I appreciate it.

**Comment by saicharanpogul** [comment]
Is there any other way i can get access of BDS?

**Comment by dee4d** [comment]
no one is maintaining this anymore. idk why they stopped it

**Comment by ljbennett62** [comment]
@Sdibyajyoti @saicharanpogul - the person who created this is leaving IBM - let me get someone else to maintain

**Comment by dee4d** [comment]
> @Sdibyajyoti @saicharanpogul - the person who created this is leaving IBM - let me get someone else to maintain

yes, please. I want this application asap to run a test model. can we have free or trial of IBM blockchain ? or we can use ibm blockchain platform instead of IBM blockchain
thanks for the reply

**Comment by FranciscoImanolSuarez** [comment]
Dear, try to contact **Sandeep Sirivolu "ssirivo@us.ibm.com",** I think you can help him, when I spoke with Rak-Joon Choi the first thing he did was contact Sandeep and he helped me a lot.

**Comment by Senapati8094** [comment]
yet no respond from the ibm. Contacted Sandeep. no reply yet

**Comment by FranciscoImanolSuarez** [comment]
Oh, I'm very sorry to read that, unfortunately I don't have another email to give you, what I can do is recommend that you enter **https://cloud.ibm.com/catalog** .Start a chat and discuss the problem you have, surely they can help you and provide a solution.
